Development

AjithBillava opened the pull request.

AjithBillava committed:
feat: added responsiveness

AjithBillava committed:
updated responsiveness

AjithBillava committed:
Update documentation.html

AjithBillava committed:
updated mobile-nav height

AjithBillava committed:
updated landing page responsiveness

AjithBillava committed:
feat: added code snippets

AjithBillava committed:
update: fixed float btn

AjithBillava committed:
updated acc to Aneena's review

1)Include get started page , so that users will get to know how to use your library.
2)Add a close icon to the modal component.
3)Change the cursor to pointer on the toast close icon.
4)Add a shade litter darker on the buttons for hover state to make the hover state more prominent.

AjithBillava committed:
Update documentation.html

AjithBillava committed:
Update documentation.html

AjithBillava committed:
Update style.css

AjithBillava committed:
Update style.css

vikasvk11 commented:
Hey Ajith, great work. Good design choices.

I would like to suggest some minor changes,
- Tone down the box shadow a little bit and use black color with alpha channel (opacity).
- In icon button, the icons are not being displayed.
- While displaying modal, nav bar is still accessible.

AjithBillava committed:
updated acc to vikas

Tone down the box shadow a little bit and use black color with alpha channel (opacity).
In icon button, the icons are not being displayed.
While displaying modal, nav bar is still accessible.
changed variables

AjithBillava committed:
Update index.html

khusharth commented:
- For the main page: change bg color to white
- documentation button color to yellow.
- documentation page: use white bg / or yellow starting with #ff....
- Navbar: make items bold.
- Change from <script src="https://artery-ui.netlify.app/documentation.js"></script> -> <script src="https://artery-ui.netlify.app/artery-ui.js"></script>
- For github: add link for your documentation site.
- Remove twitter from navbar
- component-container and component-desc should have space between them
- buttons : remove all the xs button
- For later: add card with image
- img-fluid class remove max-width: 80% --> for bluid it should have 100% width.
- For input:
![input](https://user-images.githubusercontent.com/46857858/113440939-01c26400-940b-11eb-8e6a-5e387d8e6d03.png)
- Remove min height from section class to solve the Input extra padding issue https://artery-ui.netlify.app/documentation.html#Input
- Either try to change the primary color or change from white to black fro text on yellow
- Modal: change bg to white and fix cross icon position
- Rating: just show 1 example with 2 filled and rest unfilled and show that code.

AjithBillava committed:
updated according to Kusharth's review

For the main page: change bg color to white
documentation button color to yellow.
documentation page: use white bg / or yellow starting with #ff....
Navbar: make items bold.
Change from <script src="https://artery-ui.netlify.app/documentation.js"></script> -> <script src="https://artery-ui.netlify.app/artery-ui.js"></script>
For github: add link for your documentation site.
Remove twitter from navbar
component-container and component-desc should have space between them
buttons : remove all the xs button
For later: add card with image
img-fluid class remove max-width: 80% --> for bluid it should have 100% width.
For input:
input
Remove min height from section class to solve the Input extra padding issue https://artery-ui.netlify.app/documentation.html#Input
Either try to change the primary color or change from white to black fro text on yellow
Modal: change bg to white and fix cross icon position
Rating: just show 1 example with 2 filled and rest unfilled and show that code.

AjithBillava committed:
updated : intro-desc

AjithBillava committed:
Update documentation.html

AjithBillava committed:
Update documentation.html

AjithBillava committed:
Update artery-ui.css

AjithBillava committed:
updated header and footer

AjithBillava committed:
added dark theme

AjithBillava committed:
updated modal
